packages/hint-no-broken-links/src/hint.ts

```diff
@@ -42,6 +42,9 @@ export default class NoBrokenLinksHint implements IHint {
         const requester = new Requester(options);
         const brokenStatusCodes = [404, 410, 500, 503];
 
+        /** Stores the elements with their urls which have been collected while traversing the page. */
```

Comment:
It should be `URLs` (you can see @alrra complaining about this in virtually any PR we do 😅). Can you please update it in here and any other comment instance in this file? I think our policy for `URL` in variables isn't clear. @alrra, do you want to update the code? Things like `fetchedUrls`, `urls` (eslint will complain about upper case and constructors)?

Reply:
Maybe start using `somethingURL` / `urlSomething`?

Reply:
I'm a bit confused. Should I change the spelling of "URL" only in the comments or should I rename all variables?

Reply:
> I change the spelling of "URL" only in the comments or should I rename all variables?

Both would be great! Thanks!

Reply:
Ok, no problem. And what should be the new name of the variable called `url` and `urls`? Should they be renamed to something like `uRL`?

Reply:
> variable called `url` and `urls`

@chrisguttandin They should remain the same. Thanks again!